src/app/work-item/work-item-board/work-item-board.component.html

```diff
@@ -8,43 +8,48 @@
             <span class="dib">{{lane.option}}</span>
             <span class="issueCount dib" >{{workItems | almFilterBoardList:lane.option | count}}</span>
           </div>
-          <div *ngFor='let item of workItems | almFilterBoardList:lane.option'>
-          <div class="board-card" (click)="gotoDetail(item)">
-            <aside>
-              <div class="pull-left">
-                <span class="wi-type-icon" almIcon [iconType]="item.relationships?.baseType?.data?.id"></span>
-              </div>
-              <span class="pull-left"> {{item.id}} </span>
-              <button class="pull-right btn btn-link dropdown-toggle color-grey" type="button">
-                <span class="fa fa-ellipsis-v"></span>
-              </button>
-              <ul class="dropdown-menu dropdown-menu-right">
-                <li><a class="workItemList_MoveTop">Move to Top</a></li>
-                <li><a class="workItemList_MoveBottom">Move to Bottom</a></li>
-                <li class="divider" role="presentation"></li>
-                <li><a class="workItemList_Open" href="/work-item-list/detail/429">Open</a></li>
-                <li><a class="workItemList_Delete">Delete</a></li>
-                <li><a class="workItemList_Backlog">Move to Backlog</a></li>
-              </ul>
-            </aside>
-            <aside>
-              <div class="col-md-10 col-sm-10 col-xs-9">
-                <p class="multiline-truncation">{{ item.attributes['system.title'] }}</p>
-              </div>
-              <div class="col-md-2 col-sm-2 col-xs-3 no-padding user-avatar">
-                <img *ngIf="item.relationalData?.assignees?.length"
+
+          <div class="boardlaneWrapper">
```

Comment:
@vikram-raj We have followed the camel-case convention for naming classnames and for ID we  mentioned the component name for better understanding